contributing/documentation/content_guidelines.rst

```diff
@@ -0,0 +1,181 @@
++-----+---------------------------------------------------------------------------------+
+|     | | **H2: Section Title (configuration)**                                         |
+|     | | This first H2 section is about the configuration of the feature, or the       |
+|     |   prerequisites to achieve a specific goal. To add a path, make sure you        |
+|     |   use the ``:menuselection:`` specialized directive (see link below).           |
+|     |                                                                                 |
+|     | | Example:                                                                      |
+|     | | To do so, go to ``:menuselection:`App name --> Menu --> Sub-menu```, and      |
+|     |   enable the XYZ feature.                                                       |
++-----+---------------------------------------------------------------------------------+
+|     | | **H2: Section Title (main sections)**                                         |
+|     | | Create as many main sections as you have actions or features to distinguish.  |
+|     |   The title can start with a verb, but try to avoid using "Create ...".         |
++-----+-----+---------------------------------------------------------------------------+
+|     |     | | **H3: Subsection**                                                      |
+|     |     | | Subsections are perfect for assessing very specific points. The title   |
+|     |     |   can be in the form of a question, if appropriate.                       |
++-----+-----+---------------------------------------------------------------------------+
+|     | **H2: Next Section**                                                            |
++-----+---------------------------------------------------------------------------------+
+
+.. seealso::
+   - :ref:`RST cheat sheet: headings <contributing/headings>`
+   - :ref:`RST cheat sheet: specialized directives <contributing/specialized-directives>`
+
+.. _contributing/content-images:
+
+Images
+======
+
+Adding a few images to illustrate your text helps the readers to understand and memorize your
+content. However, avoid adding too many images: it isn't necessary to illustrate all steps and
+features, and it may overload your page.
+
+.. important::
+   Don't forget to :ref:`compress your PNG files with pngquant <contributing/pngquant>`.
+
+.. _contributing/screenshots:
+
+Screenshots
+-----------
+
+Screenshots are automatically resized to fit the content block's width. This implies that
+screenshots can't be too wide, else they would appear very small on-screen. Therefore, we recommend
+to avoid to take screenshots of a full screen display of the app, unless it is relevant to do so.
+
+A few tips to improve your screenshots:
+
+#. **Zoom** in your browser. We recommend a 110% zoom for better results.
+#. | **Resize** your browser's width, either by *resizing the window* itself or by opening the
```

Comment:
Why keep the `|` ?

Reply:
dunno. He's a buddy, I promised him a gig at Odoo...
But okay, I'll tell him to find a job somewhere else.